src/userobjects/MoabSkinner.C

```diff
@@ -97,7 +101,8 @@ MoabSkinner::MoabSkinner(const InputParameters & parameters)
     _output_full(getParam<bool>("output_full")),
     _scaling(1.0),
     _n_write(0),
-    _standalone(true)
+    _standalone(true),
+    _use_displaced(getParam<bool>("use_displaced_mesh"))
```

Comment:
it's customary to put parameters which fetch references before the ones which are just setting placeholder defaults